simd/src/Kokkos_SIMD_AVX2.hpp

```diff
@@ -784,6 +791,13 @@ class simd<std::int64_t, simd_abi::avx2_fixed_size<4>> {
       _mm256_add_epi64(static_cast<__m256i>(lhs), static_cast<__m256i>(rhs)));
 }
 
+[[nodiscard]] KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION
```

Comment:
Why is this one `[[nodiscard]]`?

Reply:
Removed `[[nodiscard]]` attribute on this. I'm not why this one specifically had it.